server/services/prismic.js

```diff
@@ -167,3 +165,136 @@ export async function getCuratedList(id: string) {
 
   return curatedList;
 }
+
+async function getExhibitions(page:number = 1, pageSize:number = 40): Promise<any> {
+  const prismic = await getPrismicApi();
+  const exhibitionsList = await prismic.query([
+    Prismic.Predicates.any('document.type', ['exhibitions'])
+  ], { orderings: '[my.exhibitions.start desc]', page, pageSize });
+  return exhibitionsList;
+}
+
+async function createPromos(allResults, type) {
```

Comment:
`create` can be a dangerous word as it implies I/O where as this is conversion / parsing?